src/job/models/newJobHandler.ts

```diff
@@ -40,7 +40,7 @@ export class NewJobHandler implements IJobHandler {
           isNewTarget: true,
```

Comment:
Should check if needed on "normal" update

Reply:
we will check with asaf